cpu/cc2538/include/cpu_conf.h

```diff
@@ -65,7 +65,7 @@ extern "C" {
 /* The minimum block size which can be written is 4B. However, the erase
  * block is always FLASHPAGE_SIZE.
  */
-#define FLASHPAGE_BLOCK_SIZE    (4U)
+#define FLASHPAGE_WRITE_BLOCK_SIZE   (4U)
 /* Writing should be always 4 bytes aligned */
 #define FLASHPAGE_BLOCK_ALIGNMENT    (4U)
```

Comment:
should also be `FLASHPAGE_WRITE_BLOCK_ALIGNMENT`, right?

Reply:
Also renamed :)